src/kvstore/Part.cpp

```diff
@@ -59,7 +60,8 @@ Part::Part(GraphSpaceID spaceId,
 
 std::pair<LogID, TermID> Part::lastCommittedLogId() {
     std::string val;
-    ResultCode res = engine_->get(folly::stringPrintf("%s%d", kCommitKeyPrefix, partId_), &val);
+    auto key = folly::stringPrintf("%d%s%d", NebulaKeyType::kSystem, kCommitKeyPrefix, partId_);
```

Comment:
The format should be type + partId + "committed"

Reply:
we should move commitKey into NebulaKeyUtils

Reply:
I think we don't need to do it because the commitKey is only used in this place.